amundsen_application/static/js/components/TableDetail/TableIssues/styles.scss

```diff
@@ -0,0 +1,23 @@
+@import 'variables';
+
+.table-issues {
+
+  .issue-banner {
+    border-radius: 4px;
+    box-shadow: 0px 0px 1px 0px rgba(0, 0, 0, 0.12), 0px 2px 3px 0px rgba(0, 0, 0, 0.16);
+    height: 40px;
+    margin: 8px 0;
+    padding: 8px 16px;
+
+    &:first-child {
+      margin-top: 24px;
+    }
+  }
+  .icon-data-quality-warning {
+    background: #DB3615;
```

Comment:
Can we use a color variable here? 

Our color palette is defined in `_colors.scss` (see: $sunset60)
But we try not to reference colors directly. Instead we define a color-role in `_variables-default.scss` and then assign that to the color palette value. For example you could use `$alert-color: $sunset60;`

Reply:
i updated to what you suggested but i would love it if you re-reviewed to make sure i did the right thing. i added a variable in _variable-default.scss and then used that - is that correct?